Replicator/c4ReplicatorHelpers.hh

```diff
@@ -0,0 +1,38 @@
+//
+//  C4ReplicatorHelpers.hh
+//
+//  Copyright 2022-Present Couchbase, Inc.
+//
+// Use of this software is governed by the Business Source License included
+// in the file licenses/BSL-Couchbase.txt.  As of the Change Date specified
+// in that file, in accordance with the Business Source License, use of this
+// software will be governed by the Apache License, Version 2.0, included in
+// the file licenses/APL2.txt.
+//
+
+#pragma once
+#include "c4ReplicatorTypes.h"
+
+namespace litecore { namespace repl {
+
+    struct C4ReplParamsDefaultCollection : C4ReplicatorParameters {
+        C4ReplicationCollection defaultCollection{kC4DefaultCollectionSpec};
+
+        C4ReplParamsDefaultCollection()
+        : C4ReplicatorParameters()
+        , push(defaultCollection.push)
+        , pull(defaultCollection.pull)
+        , pushFilter(defaultCollection.pushFilter)
+        , validationFunc(defaultCollection.pullFilter)
```

Comment:
pullFilter?

Reply:
This is because you call the pullFilter in C4ReplicatorParameters validationFunc,
C4ReplicatorValidationFunction C4NULLABLE      validationFunc; 
I assumed it means pullFilter in C4ReplicationCollection,
C4ReplicatorValidationFunction C4NULLABLE      pullFilter;

Reply:
I see what you mean.